cocos/ui/UIEditBox/UIEditBox.h

```diff
@@ -483,7 +492,19 @@ namespace ui {
          * @return One of the EditBox::KeyboardReturnType constants.
          */
         KeyboardReturnType getReturnType() const;
-
+        
+        /**
+         * Set the keyboard appearance that is to be applied to the edit box.
+         * @param keyboardAppearance One of the EditBox::KeyboardAppearance constants.
+         */
+        void setKeyboardAppearance(KeyboardAppearance keyboardAppearance);
```

Comment:
Suggest to comment this method doesn't support others platforms except iOS, so as to avoid conflict when android developer try to use it